services/repository-managers/src/test/java/org/sagebionetworks/evaluation/manager/EvaluationManagerTest.java

```diff
@@ -120,56 +119,78 @@ public void setUp() throws DatastoreException, NotFoundException, InvalidModelEx
-    	when(mockPermissionsManager.hasAccess(eq(ownerInfo), any(), eq(ACCESS_TYPE.UPDATE))).thenReturn(AuthorizationStatus.authorized());
-    	when(mockPermissionsManager.hasAccess(eq(ownerInfo), any(), eq(ACCESS_TYPE.READ))).thenReturn(AuthorizationStatus.authorized());
-    	when(mockPermissionsManager.hasAccess(eq(userInfo), any(), eq(ACCESS_TYPE.READ))).thenReturn(AuthorizationStatus.accessDenied(""));
-    	when(mockPermissionsManager.hasAccess(eq(userInfo), any(), eq(ACCESS_TYPE.UPDATE))).thenReturn(AuthorizationStatus.accessDenied(""));
     	when(mockNodeDAO.getNodeTypeById(EVALUATION_CONTENT_SOURCE)).thenReturn(EntityType.project);
-    }
 
-	@Test
-	public void testCreateEvaluation() throws Exception {		
 		Evaluation clone = evaluationManager.createEvaluation(ownerInfo, eval);
 		assertNotNull(clone.getCreatedOn());
 		evalWithId.setCreatedOn(clone.getCreatedOn());
-		assertEquals("'create' returned unexpected Evaluation", evalWithId, clone);
+		assertEquals(evalWithId, clone, "'create' returned unexpected Evaluation");
 		verify(mockEvaluationDAO).create(eq(eval), eq(OWNER_ID));
 	}
 	
 	@Test
 	public void testGetEvaluation() throws DatastoreException, NotFoundException, UnauthorizedException {
-		Evaluation eval2 = evaluationManager.getEvaluation(ownerInfo, EVALUATION_ID);
+    	when(mockEvaluationDAO.get(eq(EVALUATION_ID))).thenReturn(evalWithId);
+    	evaluations= Collections.singletonList(evalWithId);
+    	when(mockPermissionsManager.hasAccess(eq(ownerInfo), any(), eq(ACCESS_TYPE.READ))).thenReturn(AuthorizationStatus.authorized());
+
+    	Evaluation eval2 = evaluationManager.getEvaluation(ownerInfo, EVALUATION_ID);
+    	
 		assertEquals(evalWithId, eval2);
 		verify(mockEvaluationDAO).get(eq(EVALUATION_ID));
 	}
 	
 	@Test
 	public void testGetEvaluationByContentSource() throws Exception {
-		List<Evaluation> qr = evaluationManager.getEvaluationByContentSource(ownerInfo, EVALUATION_CONTENT_SOURCE, 10, 0);
+
+    	evaluations= Collections.singletonList(evalWithId);
+    	when(mockEvaluationDAO.getAccessibleEvaluationsForProject(eq(EVALUATION_CONTENT_SOURCE), (List<Long>)any(), eq(ACCESS_TYPE.READ), eq(null), anyLong(), anyLong())).thenReturn(evaluations);
+
+		List<Evaluation> qr = evaluationManager.getEvaluationByContentSource(ownerInfo, EVALUATION_CONTENT_SOURCE, false, 10L, 0L);
 		assertEquals(evaluations, qr);
 		assertEquals(1L, qr.size());
 	}
 	
+	@Test
+	public void testGetEvaluationByContentSourceActiveOnly() throws Exception {
+
+    	evaluations= Collections.singletonList(evalWithId);
+    	when(mockEvaluationDAO.getAccessibleEvaluationsForProject(eq(EVALUATION_CONTENT_SOURCE), (List<Long>)any(), eq(ACCESS_TYPE.READ), anyLong(), anyLong(), anyLong())).
+    		thenReturn(Collections.EMPTY_LIST);
+
+		List<Evaluation> qr = evaluationManager.getEvaluationByContentSource(ownerInfo, EVALUATION_CONTENT_SOURCE, true, 10L, 0L);
+		assertTrue(qr.isEmpty());
```

Comment:
Please verify getAccessibleEvaluationsForProject is called (with non-null time filter in this case).